api/db/migrations/20210409154632_add_column_envoisMultiples_to_campaigns_and_add_willBeImproved_to_campaign_participations_and_update_constraint_campaign_participations_unique_userId_campaignId_willBeImproved.js

```diff
@@ -0,0 +1,39 @@
+const TABLE_NAME_CAMPAIGN = 'campaigns';
+const COLUMN_NAME_MULTIPLE_SENDING = 'multipleSendings';
+
+const TABLE_NAME_CAMPAIGN_PARTICIPATIONS = 'campaign-participations';
+const COLUMN_NAME_WILL_BE_IMPROVED = 'willBeImproved';
```

Comment:
Je propose de renommer cette colonne `isImproved` & de mettre `ISIMPROVED_COLUMN` comme nom de varaible pour être plus cohérent avec `USERID_COLUMN` et `CAMPAIGNID_COLUMN`
```suggestion
const ISIMPROVED_COLUMN = 'isImproved';
```